source/Calamari.Tests/Fixtures/PowerShell/PowerShellFixture.cs

```diff
@@ -21,12 +21,23 @@ namespace Calamari.Tests.Fixtures.PowerShell
     public class PowerShellCoreOnWindows : PowerShellFixture
     {
         protected override PowerShellEdition PowerShellEdition => PowerShellEdition.PowerShellCore;
+
+        [SetUp]
+        public void SetUp()
+        {
+            CommandLineRunner clr = new CommandLineRunner(new IgnoreCommandOutput());
+            var result = clr.Execute(new CommandLineInvocation("pwsh.exe", "--version")); 
+            if (result.HasErrors)
+                Assert.Inconclusive();
```

Comment:
Can you provide a message here? So that someone that only views the test results can see at a glance why this was an inconclusive result.

Something like:
```
Assert.Inconclusive("PowerShell Core is not installed in this machine");
```